client/packages/react-app/src/components/Web3ReactManager/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from 'react'
+import styled, { keyframes } from 'styled-components'
+import { useWeb3React } from '@web3-react/core'
+import { network } from '../../connectors'
```

Comment:
interesting...so we're using ethers on the contract side and hardhat. but using web3 for frontend? i'm guessing this was just a decision made in a CRA template?

Reply:
web3React is just a library that abstracts low level web3 library functionality. we can specify which library we use, which in this case is still ethers. relevant lines are [here](https://github.com/bazaarfinance/bazaarfinance/blob/client/client/packages/react-app/src/index.js#L3) and [here](https://github.com/bazaarfinance/bazaarfinance/blob/client/client/packages/react-app/src/index.js#L18)